docs/intro.asciidoc

```diff
@@ -44,11 +46,16 @@ var apm = initApm({
 })
 ----
 
-Alternatively, you can use a script element and the `elasticApm` global object to load and initialize the agent:
+=== Using Script Tags
+
+Add a <script> tag to the HTML page and use the `elasticApm` global object to load and initialize the agent:
+
+NOTE: Please download the latest version of JavaScript agent from `https://github.com/elastic/apm-agent-js-base/releases/latest` or
```

Comment:
Is there a reason this link and the unpgk link are formatted as code instead of links?
```suggestion
NOTE: Please download the latest version of JavaScript agent from https://github.com/elastic/apm-agent-js-base/releases/latest[GitHub] or
```